### PR TITLE
Adding gcs sync for circleci to sync bootstrap and job files

### PR DESCRIPTION
**Pull request by haroldwoo**
This pr syncs gcp_bootstrap and jobs folders to the dataproc artifacts bucket in gcs.

### CODE REVIEW

File: gcp_bootstrap/airflow_gcp.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# to be copied to gs://moz-fx-data-prod-airflow-dataproc-artifacts/airflow_gcp.sh
+# to be copied to gs://moz-fx-data-prod-airflow-dataproc-artifacts/bootstrap
```
Comment (whd) [inline gcp_bootstrap/airflow_gcp.sh]:
```suggestion
# copied to gs://moz-fx-data-prod-airflow-dataproc-artifacts/bootstrap/ by CI
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -57,6 +60,16 @@ jobs:
...
             fi
 
+  sync_gcs:
+    docker:
+      - image: google/cloud-sdk
+    working_directory: ~/mozilla/telemetry-airflow
+    steps:
+      - checkout
+      - run: |
+          gsutil rsync -r gcp_bootstrap gs://moz-fx-data-prod-airflow-dataproc-artifacts/bootstrap
```
Comment (whd) [inline .circleci/config.yml]:
```suggestion
          gsutil rsync -r -d gcp_bootstrap gs://moz-fx-data-prod-airflow-dataproc-artifacts/bootstrap
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -57,6 +60,16 @@ jobs:
...
+          gsutil rsync -r jobs gs://moz-fx-data-prod-airflow-dataproc-artifacts/jobs
```
Comment (whd) [inline .circleci/config.yml]:
```suggestion
          gsutil rsync -r -d jobs gs://moz-fx-data-prod-airflow-dataproc-artifacts/jobs
```

File: gcp_bootstrap/dataproc_init.sh [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# To be copied to gs://moz-fx-data-prod-airflow-dataproc-artifacts
+# To be copied to gs://moz-fx-data-prod-airflow-dataproc-artifacts/bootstrap
```
Comment (whd) [inline gcp_bootstrap/dataproc_init.sh]:
```suggestion
# Copied to gs://moz-fx-data-prod-airflow-dataproc-artifacts/bootstrap/ by CI
```

File: .circleci/config.yml (hunk 3 above)
Comment (haroldwoo) [reply .circleci/config.yml]: was just playing it safe. Added it back

**Pull request merged**